### ISSUE TITLE
Special character in tag

### ISSUE THREAD

**Issue by SirFlip** [shlinkio/shlink#690]
#### How Shlink is set-up

* Shlink Version: 2.0.4
* PHP Version: 7.4.3
* How do you serve Shlink: Self-hosted Apache
* Database engine used: MariaDB

#### Summary

When there are special characters in the tag name list with tag filter does not work.
Tested with * / % , . _ ö ä ü Ö Ä Ü
Does not work neither on the web-client nor on the cli
It gets saved in the DB correctly as far as I see and also diplayed correctly when the tags get listed.

#### Current behavior

Returns empty list.

#### Expected behavior

Get short-urls with that tag

#### How to reproduce

Use tag with a special character

**Comment by acelaya** [comment]
Thanks for reporting. I will try to reproduce and get it fixed.

**Comment by SirFlip** [comment]
Thank you it's a really great tool.

**Comment by acelaya** [comment]
@SirFlip I have found the issue.

When a new tag is created under any circumstance (either by creaating or editing short URLs), the tag values are filtered, triming them, converting them to lowercase, and replacing empty spaces by dashes).

However, when filtering a list of short URLs by tag, the tag values are sluggified, which does the former, plus replacing some special characters. That is not correct.

I have changed it so that the smae filtering is performed when listing short URLs, and that makes it work even when the tags have special characters.

The fix will be part of v2.1.2, which will be released today.
